front/app/modules/commercial/customizable_navbar/admin/containers/NewPageForm/index.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
```

Comment:
The form that opens when you go to `admin/settings/navigation/new-page`. Wrapper around the `PageForm` that handles submitting the form etc. Used to create new pages.

Reply:
Same here : `pages/new`